Keycloak/cloud-init.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+apt update -y
+apt install -y docker*
```

Comment:
Why not use an openSUSE Leap VM?

Reply:
Since the script is a sample for internal use, the choice between Ubuntu and openSUSE Leap may depend on what the individual is comfortable with.

Any more specific reasons why openSUSE Leap would be a better fit in this context or a dependency that Keycloak/rancher?

Reply:
> Any more specific reasons why openSUSE Leap would be a better fit in this context or a dependency that Keycloak/rancher?

No specific reason at all other than the proverb [eat your own dog food](https://en.wikipedia.org/wiki/Eating_your_own_dog_food).